xAPI.md

```diff
@@ -260,49 +260,30 @@ learning that SCORM could not enable.
+  * Developing a commonly-accepted framework for gathering, storing and exchanging this information represents the best way of achieving this.
 
-The purpose of the xAPI and the goals of this document are:
-* To maximize interoperability of systems which create, gather and store information  
-about learning experiences.
-* To provide a guide to those who want to build applications that conform to and implement this 
-specification.
+The goals of the xAPI are:
+
+* To make it easier to understand and compare learning experiences recorded in a wide variety of places.
+* To maximize interoperability of systems which create, gather, store and process information about learning experiences.
+* To provide a guide to those who want to build applications that conform to and implement this specification.
 * To provide criteria against which conformance to this specification can be tested.
 
-There are two main ways that the xAPI promotes interoperability between systems that 
-implement the specification. The first is by requiring that systems implementing the specification
-follow a consistent data structure. To that end, this specification defines a data model for various data
-objects that are transferred between systems. The most significant object within the xAPI data model is the Statement object. This
-specification defines the properties of the Statement object (including "Actor", "Verb", "Object", "Result", and "Context")
-and the rules of syntax for the values of those properties and how they are represented.
-
-The second way that the xAPI promotes interoperability is by setting out the transfer methods that must 
-be used when communicating information about learning experiences between programs that adhere to the 
-specification. As part of this, it sets out the format of requests and the expected responses. Note that the 
-systems storing or retrieving records need not be Activity Providers. LRSs can communicate with other LRSs, or systems. 
-xAPI follows the guidelines of the REST software architecture style, and as such data is tranferred via HTTP requests and
-responses. xAPI also defines security methods allowing for the trusted exchange of information between
-the Learning Record Store and trusted sources. 
-
-The Experience API is the first of many envisioned technologies that will enable
-a richer architecture of online learning and training. Authentication
-services, querying services, visualization services, and personal data services 
-are some examples of additional technologies which the Experience API is designed
-to support. While the implementation details of these services are not specified here,
-the Experience API is designed with this larger architectural vision in mind.
+The xAPI is designed pursue these goals by setting out a common framework by which organizations and systems can gather, store and exchange data about learning experiences. 
```

Comment:
"designed pursue" -> "pursues"

Gathering and storing data are out of scope of xAPI. This is the point that some of the text you've cut is trying to make clear.